site/src/components/indicator/elderRay.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: component
+title: Elder Ray
+component: indicator/algorithm/elderRay.js
+tags:
+  - frontpage
+  - playground 
+namespace: indicator
+
+example-code: |
+  // Create and apply the elder ray algorithm
+  var elderRayAlgorithm = fc.indicator.algorithm.elderRay().period(6);
+  elderRayAlgorithm(data);
+
+  // the elder ray is rendered on its own scale
+  var yScale = d3.scale.linear()
+      .domain([-5, 5])
```

Comment:
is a domain of -5 to -5 always going to be appropriate? Why not use fc.util.extent?

Actually, there's something else `fc.util.extent` should support, symmetrical domains! In this case you probably want to render a chart with a domain centred on '0', but where the maximum is computed from the data.

I'm happy to leave this example at [-5, 5] for now, but would love to see fc.util.extent extended further.